### DISCUSSION TITLE
Control both screens with touch bar or slider

### DISCUSSION THREAD

**Discussion by didiermichel** [MonitorControl/MonitorControl#712] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

There is a feature to control both screens with control keys.

### Describe the solution you'd like

It would be nice to have this same control either with:
* Common slide bar in the app menu
* or Use the touch-bar brightness control for both screens

### Describe alternatives you've considered

For now I use shortcuts.

**Comment by waydabber** [comment, marked as answer]
Hi @didiermichel

If you are using v4.0.0, please select `Use combined slider for all displays' under Preferences/App menu/Multiple displays (the option is visible only if advanced settings are enabled):

<img width="652" alt="Screen Shot 2021-10-13 at 10 26 00" src="https://user-images.githubusercontent.com/37590873/137096262-79331424-6a51-46b8-8fa5-772c808857c3.png">

If you wish the touch bar to control both the internal and external screens, you need to enable `Sync brightness changes from Built-in and Apple displays` under General.

<img width="842" alt="Screen Shot 2021-10-13 at 10 27 17" src="https://user-images.githubusercontent.com/37590873/137096446-1dfa0624-53a4-4cf8-9387-f124af6c3c5e.png">

Hope this solves the issue! Thank you!

**Comment by waydabber** [comment]
Please let me know if this solves your issue so we can follow up accordingly! Thank you! Meanwhile, I'll move this over to Discussion if there is any further question regarding these features!

**Comment by didiermichel** [comment]
Thank you, 
I didn't notice the advanced settings checkbox... 
Everything works like a charm. It's a really nice project! 

version used: v4.0.0
